ui/src/features/Editor/components/OverlayUI/components/DebugPanel/DebugPreview/components/TableViewer/FeatureDetailsOverlay.tsx

```diff
@@ -1,20 +1,31 @@
 import { ArrowLeftIcon } from "@phosphor-icons/react";
-import { memo, useMemo } from "react";
+import { memo, RefObject, useEffect, useMemo } from "react";
 
 import { IconButton } from "@flow/components";
 import { useT } from "@flow/lib/i18n";
 
 type Props = {
   feature: any;
   isOpen: boolean;
+  selectedFeature?: any;
+  featureIdMap: Map<string | number, any> | null;
+  previousSelectedFeature: RefObject<any>;
```

Comment:
I think we can move this into FeatureDetailsOverlay since we only really care about it here.